dunwoody_disaster/views/fightScreen.py

```diff
@@ -26,6 +26,10 @@ def __init__(self, player1: Character, player2: Character):
         # kick = QMovie(DD.ASSETS["P1Attack2"])
         # defense = QMovie(DD.ASSETS["P1Defense"])
         self.timer = QTimer()
+        self.init_UI()
+
+    def init_UI(self):
+        pass
```

Comment:
remove pass

Reply:
Just gonna remove that whole bit. I didn't use it anyway.